Control coverage/Feature/SubscriptionCore.md

```diff
@@ -1,14 +1,933 @@
-| <b>ControlId:</b><br>Azure_Subscription_SI_Classic_Resources<br><b>DisplayName:</b><br>Remove classic resources on a subscription.<br><b>Description: </b><br>Do not use any classic resources on a subscription. | <b>ARM API to list all resources in a Subscription:</b><br>/subscriptions/{subscriptionId}/resources?$expand=provisioningState,createdTime,changedTime&<br>api-version=2018-05-01<br><b>Property:</b><br>type<br><b>The following Classic resource types are in scope for the evaluation:</b><br>1. Microsoft.ClassicCompute/virtualMachines<br>2. Microsoft.ClassicStorage/storageAccounts<br>3. Microsoft.ClassicCompute/domainNames<br>4. Microsoft.ClassicNetwork/virtualNetworks<br>5. Microsoft.ClassicNetwork/reservedIps<br>6. Microsoft.ClassicNetwork/networkSecurityGroups<br>7. Microsoft.MarketplaceApps/classicDevServices| <b>Passed:</b><br>No Classic resources found.<br><b>Failed:</b><br>One or more Classic resources found. |
+# SubscriptionCore
+
+<!-- TOC depthfrom:2 depthto:2 -->
+
+- [Azure_Subscription_AuthZ_Remove_Deprecated_Accounts](#Azure_Subscription_AuthZ_Remove_Deprecated_Accounts)
+- [Azure_Subscription_AuthZ_Dont_Use_NonAD_Identities](#Azure_Subscription_AuthZ_Dont_Use_NonAD_Identities)
+- [Azure_Subscription_AuthZ_Dont_Use_NonAD_Identities_Privileged_Roles](#Azure_Subscription_AuthZ_Dont_Use_NonAD_Identities_Privileged_Roles)
+- [Azure_Subscription_AuthZ_Limit_ClassicAdmin_Count](#Azure_Subscription_AuthZ_Limit_ClassicAdmin_Count)
+- [Azure_Subscription_AuthZ_Remove_Management_Certs](#Azure_Subscription_AuthZ_Remove_Management_Certs)
+- [Azure_Subscription_Audit_Resolve_Azure_Security_Center_Alerts](#Azure_Subscription_Audit_Resolve_Azure_Security_Center_Alerts)
+- [Azure_Subscription_AuthZ_Custom_RBAC_Roles](#Azure_Subscription_AuthZ_Custom_RBAC_Roles)
+- [Azure_Subscription_SI_Classic_Resources](#Azure_Subscription_SI_Classic_Resources)
+- [Azure_Subscription_AuthZ_Dont_Grant_Persistent_Access](#Azure_Subscription_AuthZ_Dont_Grant_Persistent_Access)
+- [Azure_Subscription_AuthZ_Dont_Grant_Persistent_Access_RG](#Azure_Subscription_AuthZ_Dont_Grant_Persistent_Access_RG)
+- [Azure_Subscription_Config_Add_Required_Tags](#Azure_Subscription_Config_Add_Required_Tags)
+- [Azure_Subscription_Config_ASC_Defender](#Azure_Subscription_Config_ASC_Defender)
+- [Azure_Subscription_Use_Only_Alt_Credentials](#Azure_Subscription_Use_Only_Alt_Credentials)
+- [Azure_Subscription_Config_ASC_Enable_AutoProvisioning](#Azure_Subscription_Config_ASC_Enable_AutoProvisioning)
+- [Azure_Subscription_Config_ASC_Setup_SecurityContacts](#Azure_Subscription_Config_ASC_Setup_SecurityContacts)
+- [Azure_Subscription_SI_No_Billing_Activity_Trial](#Azure_Subscription_SI_No_Billing_Activity_Trial)
+- [Azure_Subscription_Configure_Conditional_Access_for_PIM](#Azure_Subscription_Configure_Conditional_Access_for_PIM)
+
+<!-- /TOC -->
+<br/>
+
+___ 
+
+## Azure_Subscription_AuthZ_Remove_Deprecated_Accounts 
+
+### DisplayName 
```

Comment:
Please rename all references to "Display Name" - this was changed in the other files very recently.